app/Http/Requests/ServerRequest.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+use Illuminate\Validation\Rule;
+
+class ServerRequest extends FormRequest
+{
+    public function rules()
+    {
+        $rules = [
+            'description'   => ['required', 'string', 'max:255'],
+            'base_url'      => ['required', 'url', 'string', 'max:255', Rule::unique('servers', 'base_url')],
```

Comment:
while it would be possible to increase it, I don't see why it would make sense. If the base url is very long, it can lead to url to long to be processed by old browsers trying to join a meeting. We write a migration to change the db fields to string, but I would leave the option open if longer urls or descriptions become necessary in the future. Are you fine with that?